apps/fabric-examples/src/pages/ButtonPage/examples/Button.Primary.Example.tsx

```diff
@@ -16,7 +16,8 @@ export class ButtonPrimaryExample extends React.Component<IButtonProps, {}> {
         <PrimaryButton
           data-automation-id='test'
           disabled={ disabled }
-        >Create account</PrimaryButton>
+          text="Create account"
+        >Text Overrides a Child String</PrimaryButton>
```

Comment:
Personally, I think it's handy to see and quite self explanatory if someone was to copy/paste it. I'm fine either way.